src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/PostActionModel.cs

```diff
@@ -75,16 +62,15 @@ internal static IReadOnlyList<IPostActionModel> ListFromJArray(JArray jObject, I
                             text = manualInstructions[i].ToString("text");
                         }
 
-                        KeyValuePair<string, string> instruction = new KeyValuePair<string, string>(text, manualInstructions[i].ToString("condition"));
-                        instructionOptions.Add(instruction);
+                        instructionOptions.Add(new ManualInstructionModel(text, manualInstructions[i].ToString("condition")));
                     }
                 }
 
                 PostActionModel model = new PostActionModel()
                 {
                     Condition = action.ToString(nameof(model.Condition)),
                     Description = actionLocalizations?.Description ?? action.ToString(nameof(model.Description)),
-                    ActionId = actionId,
+                    ActionId = action.ToGuid(nameof(ActionId)),
```

Comment:
is it a valid change?

Reply:
Yes. I just inlined the variable that wasn't used anywhere else:
https://github.com/dotnet/templating/pull/3089/files/bb688b93882c20f34ae00493bf2be4711bc1e221#diff-1efd527e44892bff897ea620e1e8b65fb4909ffd884d370a86c48f1e14a72c81L40